tests/kernel/virtio_console.pm

```diff
@@ -39,6 +39,11 @@ sub run {
 1;
 
 =head1 Configuration
+Testing virtio or svirt serial console.
```

Comment:
In what sense "Testing"? You mean something like employing/using/leveraging virtio or serial console to run the test?

Reply:
Testing virtio console and serial console backend implementation. Do you suggest some changes of phrasing?

Reply:
The original one is fine, now I understand.